Sources/Public/Animation/LottieView.swift

```diff
@@ -141,6 +141,16 @@ public struct LottieView<Placeholder: View>: UIViewConfiguringSwiftUIView {
     }
   }
 
+  /// Returns a copy of this `LottieView` updated to have the specific configuration property
+  /// applied to its represented `LottieAnimationView` whenever it is updated via the `updateUIView(...)`
+  /// or `updateNSView(...)` methods.
```

Comment:
Let's also mention that this value is applied on every SwiftUI render pass, which may be expensive for any properties that trigger side-effects when updated.

Reply:
Isn't this the case for the regular trailing-based closure `configure` ? I copied the description from there. Should we add that note there as well?

Reply:
That would be great! Although for that case you'd just say something like "the given closure is called on every SwiftUI render pass, which can be expensive for any closures that trigger side effects"